### ISSUE TITLE
Working?

### ISSUE THREAD

**Issue by luc0** [mircoc/cordova-plugin-mint#1]
This is working? thanks

**Comment by mircoc** [comment]
Yes, at the moment it work only for Android, but i will make it available for iOS too.
You only need to add the plugin, it will execute automatically on start of application and send crash report  to Splunk Mint.

**Comment by luc0** [comment]
Thanks!
